common/namespace/registry.go

```diff
@@ -349,7 +346,7 @@ func (r *registry) refreshLoop(ctx context.Context) error {
 			case r.triggerRefreshCh <- nil:
 			default:
 			}
-
+			timer.Reset(r.refreshInterval())
```

Comment:
I think there're two problems here:
1. `Reset` can work on a Stopped ticker and revive it. so if `Stop()` is called when the code is inside the for loop body, the for loop will never stop.
2. The for loop won't exit even without the `Reset`, as ticker's `Stop` method will not close its channel.